web-common/src/features/dashboards/user-preferences.ts

```diff
@@ -40,6 +40,25 @@ function localUserPreferencesActions() {
   };
 }
 
+export function getLocalUserPreferencesState(
+  exploreName: string,
+): LocalUserPreferences {
+  const localstorageUserPreference = localStorage.getItem(
+    `${exploreName}-userPreference`,
+  );
+  if (localstorageUserPreference) {
+    try {
+      return JSON.parse(localstorageUserPreference);
+    } catch {
+      // TODO
+    }
+  }
+
+  return {
+    timeZone: getLocalIANA(),
+  };
```

Comment:
I wouldn't expect this default/fallback to be included in this function. I'd expect `timeZone: getLocalIANA()` to be included in the default generic Explore state, which would later be merged with user preferences.